build/templates/_converters.py.mako

```diff
@@ -237,7 +275,8 @@ def convert_to_nitclk_session_number(item):
     raise TypeError('Unsupported type for nitclk session: {}'.format(type(item)))
 
 
-def convert_to_nitclk_session_number_list(item_list):
+# The unused parameter is required for case B520 in codegen, which expects a two-parameter converter.
+def convert_to_nitclk_session_number_list(item_list, unused):
```

Comment:
I'd rather you call `unused` -> `library_type` so caller can state the name explicitly.
Optionally you could assert the library_type if/when it makes sense to.

Reply:
Don't have `unused` parameter anymore :)